tor2web/utils/lists.py

```diff
@@ -258,7 +267,4 @@ def handleData(self, data):
             self.clear()
 
         data = json.loads(data)
-        for relay in data['relays']:
-            for ip in relay['a']:
-                if ip != '':
-                    self.add(ip)
+        self.extend( [ ip for ip in relay['a'] for relay in data['relays'] if ip ] )
```

Comment:
this is why i would prefer to keep it written with the clear nested for loops without using a one liner that make difficult the audit to who does not have clear pythonic thinking